packages/hoppscotch-backend/src/team-collection/team-collection.resolver.ts

```diff
@@ -102,6 +102,36 @@ export class TeamCollectionResolver {
     return jsonString.right;
   }
 
+  @Query(() => String, {
+    description:
+      'Returns a JSON string of all the contents of a Team Collection',
+  })
+  @UseGuards(GqlAuthGuard, GqlTeamMemberGuard)
+  @RequiresTeamRole(
+    TeamMemberRole.VIEWER,
+    TeamMemberRole.EDITOR,
+    TeamMemberRole.OWNER,
+  )
+  async exportCollectionToJSON(
+    @Args({ name: 'teamID', description: 'ID of the team', type: () => ID })
+    teamID: string,
+    @Args({
+      name: 'collectionID',
+      description: 'ID of the collection',
+      type: () => ID,
+    })
+    collectionID: string,
+  ) {
+    const collectionData =
+      await this.teamCollectionService.exportCollectionToJSONObject(
+        teamID,
+        collectionID,
+      );
+
+    if (E.isLeft(collectionData)) throwErr(collectionData.left as string);
+    return JSON.stringify(collectionData.right);
```

Comment:
Thinking, return type of both queries `exportCollectionsToJSON` and `exportCollectionToJSON` can be same.

```suggestion
    return JSON.stringify([collectionData.right]);
```